### ISSUE TITLE
`e d rbe login` randomly fails on Windows

### ISSUE THREAD

**Issue by piotrpdev** [electron/build-tools#587]
### Operating System

Windows version: `Microsoft Windows 11 Pro 10.0.22631 Build 22631`
Node.js version: `v20.10.0`
NPM version: `10.2.3`
Build tools version: `@electron/build-tools@1.2.0`
Electron RBE helper version: `v0.2.2/electron-rbe-credential-helper-windows-amd64.tar.gz`

```pwsh
PS C:\Users\peter> systeminfo | findstr /B /C:"OS Name" /B /C:"OS Version"
OS Name:                   Microsoft Windows 11 Pro
OS Version:                10.0.22631 N/A Build 22631
PS C:\Users\peter> npm list -g --depth=0
C:\Program Files\nodejs -> .\
+-- @electron/build-tools@1.2.0
+-- corepack@0.22.0
`-- npm@10.2.3

PS C:\Users\peter> yarn --version
1.22.21
```

### Expected Behavior

`e d rbe login` works every time.

### Actual Behavior

`e d rbe login` randomly fails for no apparent reason.

### To Reproduce

Try running `e d rbe login` a couple of times on Windows? I'm honestly not sure, when the command fails it does not give a very helpful error.

### Screenshots

Fails in Windows Terminal PowerShell:

https://github.com/electron/build-tools/assets/99439005/7939449e-d78c-4c3c-a064-3842e4d2b276

Fails in Command Prompt:

https://github.com/electron/build-tools/assets/99439005/46342569-083d-4c87-84a0-0b66776c5963

Fails during `e init`:

![Screenshot 2024-05-02 191958](https://github.com/electron/build-tools/assets/99439005/06628d7e-9511-4465-a5b6-f37d2f3d6f21)

### Additional Information

Also tried updating Goma, it did not help.

![Screenshot 2024-05-02 192846](https://github.com/electron/build-tools/assets/99439005/a5e2a704-b82d-4736-820d-428f6545e7ab)

```pwsh
PS C:\Users\peter> e update-goma
Writing new goma.gn file C:\Users\peter\.electron_build_tools\third_party\goma.gn
Downloading "https://dev-cdn.electronjs.org/goma-clients/8644a9443d03675cc113a26b42a20e97e41e4f4d152ff84140a9a93ea6830a08/goma-win.zip" into C:\Users\peter\.electron_build_tools\third_party\goma-win.zip
goma checksum: 8644a9443d03675cc113a26b42a20e97e41e4f4d152“84140a9a93ea6830a08
```

**Comment by dsanders11** [comment]
This could be caused by the local clock time on the machine being off by a small amount. Could you use https://time.is/ to check?

**Comment by piotrpdev** [comment]
> This could be caused by the local clock time on the machine being off by a small amount. Could you use [time.is](https://time.is/) to check?

Seems so:

![image](https://github.com/electron/build-tools/assets/99439005/92b230eb-a630-4d96-98c9-016f7ae758b9)

My machine automatically synced 4 days ago though, isn't it a bit dumb that's not good enough? Giving the user this information would also be nice.

**Comment by dsanders11** [comment]
Closed by #618, which should allow for 1m of clock skew.
